Core/src/org/sleuthkit/autopsy/texttranslation/TranslationOptionsPanelController.java

```diff
@@ -16,7 +16,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.sleuthkit.autopsy.texttranslation.ui;
+package org.sleuthkit.autopsy.texttranslation;
```

Comment:
This change will cause TranslationOptionsPanelController to be exported as it is public.

Reply:
Yes, that is intended, as we have since discussed. Thanks for your vigilance!